CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * The `pickerOptions` sub property of a color field's configuration has been merged with it's parent `options` object.
 * Reworks `inline` and `micro` UI of some fields (color, range, select). Improve global inline style.
 * Makes the range input being a number all the time instead of a string that we convert manually.
+* Command line tasks can run before the first frontend asset build without error messages.
```

Comment:
Otherwise we cannot run server side tasks that have nothing at all to do with assets before the first asset build without scaring the developer.